tools/shellcheck.sh

```diff
@@ -19,4 +19,4 @@ if ! [ -x "$(command -v shellcheck)" ]; then
 fi
 
 # TODO - fix warnings in .buildkite/run-amd-test.sh
-find . -name "*.sh" -not -path "./.buildkite/run-amd-test.sh" -print0 | xargs -0 -I {} sh -c 'git check-ignore -q "{}" || shellcheck "{}"'
+find . -name "*.sh" -not -path "./.buildkite/run-amd-test.sh" -not -path "./.venv/*" -print0 | xargs -0 -I {} sh -c 'git check-ignore -q "{}" || shellcheck "{}"'
```

Comment:
BTW, we do not have a .buildkite folder, neither the ignored script file
```suggestion
find . -name "*.sh"  -not -path "./.venv/*" -print0 | xargs -0 -I {} sh -c 'git check-ignore -q "{}" || shellcheck "{}"'
```

Reply:
I guess so

Reply:
I guess if you're going down the other path of fetching files through `git` then we don't really need to care about this change then.